docs/install.rst

```diff
@@ -4,20 +4,46 @@
 
 .. _install:
 
-==================
 Installing Bedrock
 ==================
 
-Installation
-------------
+There are two primary methods of installing bedrock: Docker and Local. Whichever you choose you'll start by getting the source::
 
-These instructions assume you have `git` and `pip` installed. If you don't have `pip` installed
-(you probably do) you can install it with the instructions in `the pip docs <https://pip.pypa.io/en/stable/installing/>`_.
+    $ git clone --recursive git://github.com/mozilla/bedrock.git
+    $ cd bedrock
 
-Start by getting the source::
+Now configure the application to run locally by creating your local settings environment file
+(You shouldn't need to customize anything in there yet)::
 
-    $ git clone git://github.com/mozilla/bedrock.git
-    $ cd bedrock
+    $ cp .env-dist .env
+
+After these basic steps you can choose your install method below. Docker is the easiest and recommended way, but local is also possible
+and may be preferred by people for various reasons.
+
+Docker Installation
+-------------------
+
+.. note::
+
+    This method assumes you have `Docker installed for your platform <https://www.docker.com/community-edition#/download>`_.
+    If not please do that now or skip to the ``Local Installation`` section.
+
+This is the simplest way to get started developing for bedrock. If you're on Linux or Mac (and possibly Windows 10 with the
+Linux subsystem) you can run a script that will pull our production docker images and use those to build developer ones and
+start the servers you need to serve bedrock locally::
+
+    $ bin/docker-start.sh
```

Comment:
I ran this locally and all the build part worked, but when I go to view the home page locally I get the following DB error: https://gist.github.com/alexgibson/54f7ff0b053dfdd898d1e7b697dfcd25

Note: other pages that don't have a newsletter such as `/firefox` work as expected.

Reply:
I believe this is due to your local database file not being up-to-date. I believe most users of the docker method won't have any local db, but it's possible. I'll see what I can do.